servlet/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    api 'javax.servlet:javax.servlet-api:4.0.1'
+    implementation 'org.apache.commons:commons-lang3:3.10'
```

Comment:
What is this for?

Reply:
I want to use `org.apache.commons.lang3.tuple.Pair` at class `ServletUrlMapper` ,  we can write the same class for this

Reply:
Could just use `Map.Entry` instantiated with `Maps.immutableEntry()`?